WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -397,6 +402,7 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
         configureRichText()
         configureDiscoverAttribution()
         configureTag()
+        configureRelatedPosts()
```

Comment:
Well, not really. Its expected that the segue can be performed prior to a ReaderPost being assigned, and the configuration needs to happen only when the post is available.  An example of this is when the detail is instantiated and passed a siteID and postID rather than a post object.  It will take a moment to fetch the post from the API.   So for these reasons the config is called here.